zerver/tests/test_invite.py

```diff
@@ -2829,6 +2880,68 @@ def check_user_able_to_register(self, email: str, invite_link: str) -> None:
 
         mail.outbox.pop()
 
+    def test_welcome_bot_custom_message_from_normal_account(self) -> None:
+        realm = get_realm("zulip")
+        full_members_system_group = NamedUserGroup.objects.get(
+            name=SystemGroups.FULL_MEMBERS, realm=realm, is_system_group=True
+        )
+        self.login("hamlet")
+        welcome_bot_custom_message = "Welcome to Zulip!"
+        result = self.client_post(
+            "/json/invites/multiuse",
+            {
+                "welcome_bot_custom_message": welcome_bot_custom_message,
+            },
+        )
+        self.assert_json_error(result, "Insufficient permission")
```

Comment:
I think there is no need to check this case, as we might be already testing the case where user does not have permission to create multiuse invites.